### PR TITLE
getting ready for python 3 - part 2

### PR DESCRIPTION
**Pull request by adhintz**
* Change from print statement to print() function.
python 2 to 3: https://portingguide.readthedocs.io/en/latest/builtins.html#the-print-function

* Replace xrange with range
python 2 to 3: https://portingguide.readthedocs.io/en/latest/iterators.html#new-behavior-of-range
Where there's a chance of performance issues with switching from the python 2 xrange to python 2 range, I've imported `from six.moves import range` to preserve the iterator behavior.
However in places where there's not a performance issue, I've left out the six import so that there's less code churn after switching to python 3.

* Explicitly use truncating division in one more place
python 2 to 3: https://portingguide.readthedocs.io/en/latest/numbers.html#division

I've tested this PR with the docker test and a real drive.

### CODE REVIEW

**Comment by geohot** [comment]
We are currently building a new /usr with python 3.7 included that will ship with NEOS 10.

Whether we migrate comes down to is CPU usage. Python 3 = more, no. Python 3 = less, yes. Python 3 = same, maybe.

After we ship new /usr, if you could port and benchmark a single service and compare, that would be great. controlsd uses less CPU = we will commit to Python 3.

File: selfdrive/can/tests/test_packer_toyota.py [hunk 1]
```diff
@@ -8,6 +8,7 @@
   create_fcw_command, create_ui_command
 )
 from common.realtime import sec_since_boot
+from six.moves import range
```
Comment (geohot) [inline selfdrive/can/tests/test_packer_toyota.py]: I don't like six, why do we need it here?

File: selfdrive/boardd/tests/test_boardd_api.py [hunk 2]
```diff
@@ -7,20 +7,21 @@
 from common.realtime import sec_since_boot
 from cereal import log
 import unittest
+from six.moves import range
```
Comment (geohot) [inline selfdrive/boardd/tests/test_boardd_api.py]: For tests, I don't think creating the list matters

**Comment by geohot** [comment]
Can we remove six? I think a lot of those ranges are in tests. If so, we are good to merge.

File: selfdrive/boardd/tests/test_boardd_api.py (hunk 2 above)
Comment (adhintz) [reply selfdrive/boardd/tests/test_boardd_api.py]: Agreed, removing six.

File: selfdrive/can/tests/test_packer_toyota.py (hunk 1 above)
Comment (adhintz) [reply selfdrive/can/tests/test_packer_toyota.py]: Removing six.

**Comment by adhintz** [comment]
Thanks for the feedback. I've rebased and removed six.

**Comment by adhintz** [comment]
Want to merge this PR, or want me to benchmark controlsd with python3 first?

**Comment by geohot** [comment]
@rbiasini @pd0wm If this doesn't make things slower in tests, can we merge it?

**Comment by rbiasini** [comment]
Thanks @adhintz , merged in internally due to the minor conflicts. Will be in next release.

Since we are now close to end of Python 2 support, it makes sense for us to do a big company push and move to Python 3 all at once internally. Thanks for the changes you made so far.

**Pull request closed without merging**